clientapp/CMakeLists.txt

```diff
@@ -42,6 +42,11 @@ target_include_directories(vcmiclient
 	PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}
 )
 
+if(ENABLE_VIDEO)
+	target_compile_definitions(vcmiclient PRIVATE ENABLE_VIDEO)
+endif()
```

Comment:
no longer needed in this file I guess?

Reply:
Still needed. See changes in `EntryPoint.cpp` - used to construct video handler if ffmpeg support is on.

Reply:
I wonder why it was only inside WIN32 then...